src/ui/flutter_app/lib/services/environment_config.dart

```diff
@@ -22,13 +22,13 @@
 class EnvironmentConfig {
   const EnvironmentConfig._();
 
-  /// gets weather we build for a monkey test
-  static const monkeyTest = bool.fromEnvironment('monkey_test');
+  /// Get whether we build for Monkey/Appium testing
```

Comment:
https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs

🙃 it should be retunred to `Gets`

Reply:
It should be fine whether to add `s`. Added back `s`.